manifests/package.pp

```diff
@@ -0,0 +1,5 @@
+class mcollective::package {
+  packages { $mcollective::package_name:
```

Comment:
Thanks looks good, lets squash the commits to one - only the latest and remove the blanks here at the same time so its `package{$mcollective::package_name:...}` - note fixing the mistake of `packages` to `package` at the same time

Reply:
Good catch, fixed